api/types.go

```diff
@@ -165,9 +168,10 @@ type TokenResponse struct {
 }
 
 type GenerateResponse struct {
-	Model     string    `json:"model"`
-	CreatedAt time.Time `json:"created_at"`
-	Response  string    `json:"response"`
+	Model      string    `json:"model"`
+	MultiModal bool      `json:"multi_modal"`
```

Comment:
Seems like this `MultiModel` response flag may not be needed. The client knows when it is sending a multimodel request, what was the use-case for this?